src/Bijectors.jl

```diff
@@ -54,29 +54,28 @@ export  TransformDistribution,
         logpdf_with_trans,
         isclosedform,
         transform,
+        transform!,
         with_logabsdet_jacobian,
+        with_logabsdet_jacobian!,
         inverse,
-        forward,
         logabsdetjac,
+        logabsdetjac!,
         logabsdetjacinv,
         Bijector,
         ADBijector,
         Inverse,
-        Composed,
-        compose,
         Stacked,
         stack,
         Identity,
         bijector,
         transformed,
         UnivariateTransformed,
         MultivariateTransformed,
-        logpdf_with_jac,
-        logpdf_forward,
         PlanarLayer,
         RadialLayer,
-        CouplingLayer,
-        InvertibleBatchNorm
+        Coupling,
+        InvertibleBatchNorm,
+        elementwise
```

Comment:
Hmm... I wonder if we could reuse some existing functionality in the ecosystem here. And/or if there is a shorter name. Regarding the first point, e.g., `Transducers.Map` seems similar?

Reply:
I'm happy to take suggestions, but I'm also okay with `elementwise`, so IMO this shouldn't hold this PR back.

Not too big of a fan to depend use `Transducers.Map` though; seems like unnecessary complexity just to make `Base.Fix1(broadcast, f)`.

Reply:
No, I don't want to depend on Transducers either. Unfortunately, it seems we can't just define a curried version
```julia
Base.map(t::Transform) = ... 
```
or
```julia
Base.broadcast(t::Transform) = ...
```
since we would like to use `elementwise` also for functions such as `exp`?